activemq-amqp/pom.xml

```diff
@@ -77,10 +77,13 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <!-- using it for Jetty's JNDI context to work /w Joram tests. -->
-      <groupId>org.eclipse.jetty.aggregate</groupId>
-      <artifactId>jetty-all</artifactId>
-      <classifier>uber</classifier>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-server</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Is this actually needed? If the comment is accurate (may not be), seems as likely only the lower jetty-jndi dep is actually needed.

Reply:
The answer is its not strictly needed, as its a transitive dependency of jettys websocket-server dep that already is used in the module, but is used and so isnt really doing any harm adding it.

Reply:
Let me give you some background:
- jetty 9.4.45 contains new CVE fixes
- just updating to jetty 9.4.45 using jetty-all uber jar won't work as jetty broke the uber jar
So, that's why I did both changes in the same PR.

Reply:
I wasnt aware since it wasnt said anywhere that the uber jar was broken. Makes sense then.